bin/retrieve_eload_from_lts.py

```diff
@@ -0,0 +1,149 @@
+    # but exclude compressed vcf files (the ones ending with .vcf.gz)
+    if os.path.exists(dir_path):
+        file_list = []
+        for root, directories, files in os.walk(dir_path):
+            for file in files:
+                if file[-3:] == '.gz' and file[-7:] != '.vcf.gz':
+                    file_list.append(os.path.join(root, file))
+        return file_list
+    else:
+        return []
+
+
+def uncompress_files(files_to_uncompress):
+    for file in files_to_uncompress:
+        command = f"gzip -f -dv {file}"
+        command_utils.run_command_with_output('Uncompress file', command)
+
+
+def get_project_from_eload_config(retrieved_dir, archive_name):
+    archive_name = archive_name.rstrip('.tar')
+    config_file_path = os.path.join(retrieved_dir, archive_name, f'.{archive_name}_config.yml')
+
+    if os.path.isfile(config_file_path):
+        with open(config_file_path, 'r') as eload_config:
+            eload_content = yaml.safe_load(eload_config)
+            try:
+                project_acc = eload_content['brokering']['ena']['PROJECT']
+                return project_acc
+            except:
+                logging.warning(f'Project accession could not be retrieved for ELOAD {archive_name}')
+
+
+def update_path_in_eload_config(retrieved_dir, archive_name):
+    archive_name = archive_name.rstrip('.tar')
+    config_file_path = os.path.join(retrieved_dir, archive_name, f'.{archive_name}_config.yml')
+
+    if os.path.isfile(config_file_path):
+        with fileinput.input(config_file_path, inplace=True) as file:
+            for line in file:
+                new_line = line.replace('/nfs/production3', '/nfs/production/keane')
+                print(new_line, end='')
+
+
+def retrieve_archive(archive_path, retrieval_output_path, files_dirs_to_retrieve=''):
+    if os.path.exists(archive_path):
+        command = f"tar -xf {archive_path} -C {retrieval_output_path} {files_dirs_to_retrieve}"
+        command_utils.run_command_with_output('Retrieve files/dir from tar', command)
+
+
+def get_files_or_dirs_to_retrieve_from_archive(archive):
+    files_dirs_to_retrieve = []
+    if 'retrieve_dirs' in archive and archive['retrieve_dirs']:
+        files_dirs_to_retrieve.extend(archive['retrieve_dirs'])
+    if 'retrieve_files' in archive and archive['retrieve_files']:
+        files_dirs_to_retrieve.extend(archive['retrieve_files'])
+
+    return ' '.join(files_dirs_to_retrieve)
+
+
+def retrieve_eloads(eloads_archive_info, eloads_archive_dir, projects_archive_dir):
+    eloads_output_path = eloads_archive_info['eloads_output_path']
+    projects_output_path = eloads_archive_info['projects_output_path']
+
+    if not os.path.exists(eloads_output_path):
+        os.makedirs(eloads_output_path)
+
+    logging.info(f"Retrieving Eloads")
+    for eload_archive_info in eloads_archive_info['eload_archives']:
+        ## Retrieve eload
+        eload_name = eload_archive_info['name']
```

Comment:
i'm a bit confused as to why that name is buried inside the config file. Would it be better to get it from the command line ?